### DISCUSSION TITLE
Note Scrollbar

### DISCUSSION THREAD

**Discussion by DevAnyKR** [moneymanagerex/moneymanagerex#6778] [Ideas]
![image](https://github.com/moneymanagerex/moneymanagerex/assets/110871727/8b740d98-33ef-4a4c-a817-7a8f73358f24)

I don't want to open the dialog or adjust the split position to see the content.

Please insert the scrollbar.

**Comment by renato-mmex** [comment]
The window can be reduced in size by dragging

<img width="203" alt="image" src="https://github.com/moneymanagerex/moneymanagerex/assets/76920138/57f82cb7-c1e4-4200-bf9e-e52fa9d67562">

**Reply by DevAnyKR** [reply]
I don't understand why you are responding to this so far. 🤣🤣🤣
I wasn't asking how to adjust the size, I was writing an idea opinion on the function.

**Reply by renato-mmex** [reply]
I was just expressing my opinion / idea to your idea and that's what you wanted when you wrote this post under discussions. 

I have minimised the window to one line and if I want to see more, I make the window bigger. 
If you were to set the window to 2 lines and introduce a scroll bar, for example, you would have to scroll constantly for larger texts.
If you suggest a scroll bar in addition to the current solution, this could increase convenience.

**Comment by Macaulay1234** [comment]
Yes if possible
